README.md

```diff
@@ -141,3 +141,20 @@ Import the override file into the main `app.scss`
 [build-status-link]: https://dev.azure.com/knoxville-utilities-board/ember-nrg-ui/_build/latest?definitionId=1&branchName=master
 [npm-badge-img]: https://badge.fury.io/js/ember-nrg-ui.svg
 [npm-badge-link]: http://badge.fury.io/js/ember-nrg-ui
+
+### Displaying Environment In App Bar
+
+By default, every environment will be displayed in the App Bar except for `prod`.
+To change this, add an array property called `productionEnvironments` to  the `ENV` variable in the `config/environment.js` file for your app.
+For example, if your production environments are `prd` and `prod` the `environment.js` file would look something like this:
+
+```javascript
+module.exports = function(/* environment, appConfig */) {
+  const ENV = {
+    ...
+    productionEnvironments: ['prd', 'prod'] // <- Look Here
```

Comment:
I have one last thing. Can you make it so this is nested under a object? `ember-nrg-ui`?

This provides a namespace for the options as to not conflict with other addons and/or ember builds.

This seems to be the common practice with many addons.